cmd/zpool/zpool_main.c

```diff
@@ -676,6 +676,20 @@ zpool_do_add(int argc, char **argv)
 		return (1);
 	}
 
+	/* unless manually specified use "ashift" pool property for new vdevs */
+	if (!nvlist_exists(props, ZPOOL_CONFIG_ASHIFT)) {
```

Comment:
Say someone made a pool without setting it and then wants to set it. Right now, it isn't a big deal because the property is meaningless. Give it a meaning and it becomes something that needs to be able to be changed.